validator/validator_test.go

```diff
@@ -90,12 +90,10 @@ testdata/invalid_env.proto:24:16: "org.federation.Invalid" message does not exis
                     ^
 `},
 		{file: "invalid_multiple_env.proto", expected: `
-testdata/invalid_multiple_env.proto:12:9: "org.federation.InlineEnvService": multiple services within the same package ("org.federation") cannot use the env option
-12:      env {
-             ^
-testdata/invalid_multiple_env.proto:48:9: "org.federation.RefEnvService": multiple services within the same package ("org.federation") cannot use the env option
-48:      env { message: "Env" }
-             ^
+testdata/invalid_multiple_env.proto:56:1: environment variable "ccc" has different types across services: service: InlineEnvService, type: int64, repeated: false, service: RefEnvService, type: int64, repeated: true
+environment variable "eee" has different types across services: service: InlineEnvService, type: message, repeated: false, service: RefEnvService, type: message, repeated: false
```

Comment:
The current error message does not show which field does not match, but I could not think of an easy way to do that...

Reply:
I think it's sufficient to just display the name of the Env and the different Service names.